### ISSUE TITLE
Import error in hooks

### ISSUE THREAD

**Issue by teone** [apiaryio/dredd-hooks-python#17]
I'm trying to create a `helpers` module to setup the db in my tests and import in into my `hooks.py` file.

This is the folder structure:
```
helpers
- __init__.py
- subscribers.py
hooks.py
```
the subscriber module is:
```
import os
import sys
sys.path.append("/opt/xos")
os.environ.setdefault("DJANGO_SETTINGS_MODULE", "xos.settings")
import django
from core.models import *
#from hpc.models import *
from services.cord.models import *
django.setup()

def createTestSubscriber():
    # deleting all subscribers
    for s in CordSubscriberRoot.objects.all():
        print(s.name)
        s.delete(purge=True)

    # creating the test subscriber
    subscriber = CordSubscriberRoot(name='Test Subscriber 1')
    subscriber.save()
    print "Subscriber Created"
```

I was trying to import it into `hooks.py` as:
`from helpers import subscriber`

but I keep getting: `ImportError: No module named helpers`,
googling around it seems to be the right syntax, any idea of what I’m missing?

Thanks in advance

**Comment by w-vi** [comment]
@teone thanks for the report, what version of python  are you using?

**Comment by teone** [comment]
`Python 2.7.6`

**Comment by w-vi** [comment]
Sorry for not repsonding got busy by other tasks. Is it still valid or have you found a solution?

**Comment by DomhnallBoyle** [comment]
I'm having the same problem trying to import a function from another module into the dredd hooks module. It keeps saying 'No module named ...' even though the module is in the same directory as the dredd hooks one

**Comment by w-vi** [comment]
@DomhnallBoyle Are you on Python 2.7 as well?

**Comment by DomhnallBoyle** [comment]
Yes 2.7.6

**Comment by w-vi** [comment]
Cheers, will look into it.

**Comment by DomhnallBoyle** [comment]
Thank-you

**Comment by honzajavorek** [comment]
@teone @DomhnallBoyle could you please confirm this has been fixed in #24? Thanks!

**Comment by DomhnallBoyle** [comment]
Yes works great, thank-you!
